app/src/app/[lng]/onboarding/setup/page.tsx

```diff
@@ -294,7 +294,7 @@ function SetupStep({
                     required: t("select-city-required"),
                   })}
                   autoComplete="off"
-                  onFocus={() => setOnInputClicked(true)}
+                  onKeyUp={() => setOnInputClicked(true)}
```

Comment:
Why did you make this change? If anything I think both listeners would need to be connected, so perhaps keep the original line and just add this?

Reply:
I changed the event listener from Onfocus to onKeyUp because the input Text field wasn't that reactive with the OnFocus Listener. The onFocus Listener was only fired when the input text field was focused.  In my experience, I had to type a city name, click outside the input field and, click back before it fetched the list of cities.  

Nice suggestion, Both listeners could be connected. I will add the onFocus too.